Automattic/vip
# Add notifications commands to package.json

## Description
Adding the new notifications commands to `package.json#bin` so as to be able to invoke them as - 
`vip @app.environment notifications [add | list]` after running `npm link` ([doc](https://github.com/Automattic/vip-cli/blob/develop/CONTRIBUTING.md#adding-commands)).

Did not add it for the `notifications get` and `notifications edit/update` commands as I couldn't get them working.

## Steps to Test
1. Check out PR.
2. Run `npm run build`
3. Run `VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip config notifications`. It'll fail with `Error: spawn vip-command ENOENT`
4. Run `npm link` so that `arg` knows how to spawn the command locally. Now `VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip config notifications` should work
